### PR TITLE
Fixed build errors stemming from camelCase in templates

### PR DESCRIPTION
**Pull request by ksnyde**
Fixed build errors stemming from _camelCase_ in templates where VueJS expects _dasherized_ variables

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "search.exclude": {
```
Comment (ksnyde) [inline .vscode/settings.json]: I think there are enough vs-code users in the NativeScript community that some sensible defaults for file exclusion are reasonable to include in the repo too. What do you think?

File: app/components/Groceries/GroceryList.vue [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <template>
   <list-view :items="items" :row="row" :class="{'visible': listLoaded}"
-    @itemLoading="makeBackgroundTransparent(event)">
-    <template scope="item">
+    @item-loading="makeBackgroundTransparent(event)">
+    <template slot-scope="item">
```
Comment (ksnyde) [inline app/components/Groceries/GroceryList.vue]: Apparently `scope` is now deprecated in favor of `slot-scope`

**Comment by ksnyde** [comment]
@tralves please note that I *AM* getting a runtime error currently:

> file:///app/components/Groceries/Groceries.vue:1: JS ERROR SyntaxError: Unexpected token '<'

I'm still very new to NS and VueJS so trying to put my head around this error. The changes made to this file were all quite controlled and actually since it's complaining about the first line (aka, the "template" tag) then it seems like it's something else.

File: app/main.js [hunk 3]
```diff
@@ -28,7 +28,7 @@ const router = new VueRouter({
     {
       path: '/login',
       component: Login,
-      meta: { hidenActionBar: true }
+      meta: { hiddenActionBar: true }
```
Comment (ksnyde) [inline app/main.js]: this meta info appears not to be being used but it was misspelled so just fixed that.

Comment (tralves) [reply app/main.js]: Yup. I was using that to show/hide the action bar. I will delete it soon.

**Comment by tralves** [comment]
Thanks for the PR!

**Comment by ksnyde** [comment]
Sure. BTW, Are you getting the same error I mentioned above in the `Groceries.vue` file? Feels like a phantom error, maybe something with deps or build process. Anyway, very interested to know whether yours is working.

**Comment by ksnyde** [comment]
Paranoid that my attempt to "help" has resulted in an error. Your repo is such a good source of learning for those *vue* inclined.

**Comment by tralves** [comment]
Thanks :)

I will have to look into it. There are several things in the build process that are still bugging me. I want to give another shot with webpack because these errors (and the errors in the other issue you created) are weird and won't go away.

I had to ignore the warnings if I wanted to ever finish this app but yeah... it sucks. :(

**Pull request merged**